<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PatMan - Patient Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="patientsTable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>6</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Vorname</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Alter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Größe</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Gewicht</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>BMI</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="crudHorizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="nameVerticalLayout">
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="givenNameVerticalLayout">
          <item>
           <widget class="QLabel" name="givenNameLabel">
            <property name="text">
             <string>Vorname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="givenNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ageVerticalLayout">
          <item>
           <widget class="QLabel" name="ageLabel">
            <property name="text">
             <string>Alter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ageSpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="sizeVerticalLayout">
          <item>
           <widget class="QLabel" name="sizeLabel">
            <property name="text">
             <string>Größe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sizeSpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="weightVerticalLayout">
          <item>
           <widget class="QLabel" name="weightLabel">
            <property name="text">
             <string>Gewicht</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="weightSpinBox">
            <property name="maximum">
             <double>300.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addPushButton">
          <property name="text">
           <string>Hinzufügen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updatePushButton">
          <property name="text">
           <string>Ändern</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePushButton">
          <property name="text">
           <string>Löschen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
